test/BRP-Bevragen-v2-postman-collection.json

```diff
@@ -0,0 +1,2795 @@
+{
+	"info": {
+		"_postman_id": "7405d556-a548-495f-bc3c-4b655d7d4589",
+		"name": "Bevragen Personen v2",
+		"description": "API voor het bevragen van personen uit de basisregistratie personen (BRP), inclusief de registratie niet-ingezeten (RNI). Met deze API kun je personen zoeken en actuele gegevens over personen, kinderen, partners en ouders raadplegen.\n\nGegevens die er niet zijn of niet actueel zijn krijg je niet terug. Heeft een persoon bijvoorbeeld geen geldige nationaliteit, en alleen een beëindigd partnerschap, dan krijg je geen gegevens over nationaliteit en partner.\n\nZie de [Functionele documentatie](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v1.1.0/features) voor nadere toelichting.\n",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Zoek met nummeraanduiding identificatie",
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"fields\": \"<string>\",\n    \"type\": \"<string>\",\n    \"gemeenteVanInschrijving\": \"<string>\"\n}",
```

Comment:
Bij de andere zoektypes heb je wel type ingevuld met de juiste waarde en de andere parameters met een voorbeeldwaarde. 
De "name" hier  is Zoek met nummeraanduiding identificatie", maar de parameter "nummeraanduidingIdentificatie" is nu juist niet in de body opgenomen. 

Bijvoorbeeld:
{
  "type": "ZoekMetNummeraanduidingIdentificatie",
  "fields": "burgerservicenummer,naam,verblijfplaats",
  "nummeraanduidingIdentificatie": "0599200000219678",
  "gemeenteVanInschrijving": "0599"
}